config/cache.php

```diff
@@ -14,7 +14,7 @@
 
     $services
         ->defaults()
-        ->private();
+        ->public();
```

Comment:
This is for testing that using builder with `withCache()` loads this file, i.e. the aliases are set.